octoprint_backupscheduler/static/js/backupscheduler.js

```diff
@@ -4,33 +4,107 @@
  * Author: jneilliii
  * License: MIT
  */
-$(function() {
+$(function () {
     function BackupschedulerViewModel(parameters) {
         var self = this;
 
         self.settingsViewModel = parameters[0];
-        self.cert_saved = ko.observable(false);
-        self.show_time = ko.pureComputed(function(){
-            return self.settingsViewModel.settings.plugins.backupscheduler.backup_daily() || self.settingsViewModel.settings.plugins.backupscheduler.backup_weekly() || self.settingsViewModel.settings.plugins.backupscheduler.backup_monthly()
-        })
+        self.sendTestEmailRunning = ko.observable(false);
 
-        self.onSettingsBeforeSave = function(){
-            if(!self.settingsViewModel.settings.plugins.backupscheduler.daily.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)){
+        //send retained notification
+        self.onStartupComplete = function () {
+            if (self.settingsViewModel.settings.plugins.backupscheduler.notification.retainedNotifyMessageID() !== "") {
+                self.onDataUpdaterPluginMessage(plugin, { notifyMessageID: self.settingsViewModel.settings.plugins.backupscheduler.notification.retainedNotifyMessageID() })
+            }
+        }
+
+        self.onSettingsBeforeSave = function () {
+            if (!self.settingsViewModel.settings.plugins.backupscheduler.daily.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)) {
                 self.settingsViewModel.settings.plugins.backupscheduler.daily.time('00:00')
             }
-            if(!self.settingsViewModel.settings.plugins.backupscheduler.weekly.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)){
+            if (!self.settingsViewModel.settings.plugins.backupscheduler.weekly.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)) {
                 self.settingsViewModel.settings.plugins.backupscheduler.weekly.time('00:00')
             }
-            if(!self.settingsViewModel.settings.plugins.backupscheduler.monthly.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)){
+            if (!self.settingsViewModel.settings.plugins.backupscheduler.monthly.time().match(/([01]?[0-9]|2[0-3]):[0-5][0-9]/)) {
                 self.settingsViewModel.settings.plugins.backupscheduler.monthly.time('00:00')
             }
         }
+
+        // receive data from server
+        self.onDataUpdaterPluginMessage = function (plugin, data) {
```

Comment:
It's best to exit early out of this function if plugin !== plugin_identifier. Should add that to the start of the function, otherwise you risk picking up messages from other plugins using the same "data" structure. 

```js
		self.onDataUpdaterPluginMessage = function (plugin, data) {
			if (plugin !== "backupscheduler") return;
```